cves/CVE-2017-12616.yml

```diff
@@ -4,27 +4,27 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Dirsclosure
```

Comment:
*Disclosure

Reply:
its a made up nickname that combines the word directory and disclosure (as in information disclosure)

Reply:
Oh, sorry. Thought it was a typo.